AprilSylph/XKit-Rewritten
# AccessKit Disable Gifs: Fix hover-to-animate in post editor trail

Is this the most pointless PR of all time? I think it probably is.

### Description


Ensures that the Disable Gifs behavior of unfreezing an entire photoset when any of it is hovered is applied to the trail items shown in the post editor when reblogging.

### Testing steps


Find a post with multiple gifs in a photoset (I search for "gif"). Press reblog on it. Mouse over the photoset in the trail within the editor. All of the gifs in the photoset should animate.
